collect_app/src/main/java/org/odk/collect/android/utilities/ColorPickerDialog.java

```diff
@@ -146,7 +148,7 @@ public boolean onInterceptTouchEvent(MotionEvent event) {
             // Initializes the Paint that will draw the View
             paint = new Paint(Paint.ANTI_ALIAS_FLAG);
             paint.setTextAlign(Paint.Align.CENTER);
-            paint.setTextSize(12);
+            paint.setTextSize(12 * scaleFactor);
```

Comment:
This might be a good opportunity to eliminate all the magic numbers (12, 10, 40, 50, 306, 316, 138, 356, 74, 340, 266, 276, etc.), replacing them with constants.

Reply:
To illustrate, I’ve made a few constants here:
https://github.com/dcbriccetti/collect/commit/8fc8a77e28d5f18f00bc7963ae485d0d80734f8a

I’m starting to think, though, that it isn’t worth spending too much time on our home-grown version of something as common as a color picker, especially when there seems to be scores of them already written and debugged.

Reply:
@dcbriccetti I don't get this: `NUM_HUES * 2 * DIALOG_MARGIN`. Don't you get a very large number for `DIALOG_WIDTH`?

Reply:
It’s identical if you set NUM_HUES to 256 (if you don’t change from 256 to 360 hues).

Reply:
I’ve updated my example changes here:
https://github.com/dcbriccetti/collect/commit/41aa32ab38bd5bd903bb80de12d13e0da8d3f428